### ISSUE TITLE
Crossing the finish line

### ISSUE THREAD

**Issue by benmontet** [benmontet/k2-characterization#5]
I think the following is all we need:
- [x] **Tim**: finish stellar properties where they are currently broken
- [ ] **Dan**: Table and subsection on planet properties
- [x] **Tim**: False positive probabilities data and subsection.
- [x] **Brendan**: Contrast curves for 8 systems
- [ ] **Ben**: Abstract, introduction, finish results, and conclusion.
- [ ] **Anyone**: Minor crap to check throughout, usually noted in redline.

I'd love to have this out to everyone as soon as possible and submit by the end of the weekend, before we get beaten to too many other confirmations. 

To that end, I think we should proceed as if we will not have any contrast curves, and be pleasantly surprised if that changes.

Let me know what I can do to help each of you tick off your boxes as efficiently as possible.

**Comment by timothydmorton** [comment]
New stellar properties samples posted [here](http://www.astro.princeton.edu/~tdm/k2/starmodels).  Also included are triangle plots, both of the physical properties of each star, and how the predictions of observed properties according to the MCMC samples match the true observed properties themselves.

**Comment by dfm** [comment]
@benmontet: what is the full list of columns that you want in that table? I remember you mentioning something about surface temperature or something... I think that's the one thing that I don't know how to compute...

**Comment by benmontet** [comment]
I think the following are appropriate:

Planet number, period, maximum allowed secondary (maybe in units of the transit depth?), planet radius, a/R_Star, T_equilibrium, FPP.

I presume @timothydmorton will drop in FPP, and I believe you are set up well for everything else? T_eq is given as T_eff,star \* (2_a/R_star)_*(-0.5).

**Comment by dfm** [comment]
or... T_eff \* sqrt(R_star / (2 \* a)) ?

**Comment by benmontet** [comment]
That too!

**Comment by timothydmorton** [comment]
Initial FPP results in fpp branch, fpp/fppresults.txt.  Haven't gone through to make sure these all make sense, but highlights include the following likely (>80%) FPs:

201257461.1
201505350.1 (first of two candidates...)
201565013.1
201569483.1
201649426.1
201779067.1

**Comment by benmontet** [comment]
I'll look through these tomorrow, but Armstrong+ claim hour-ish TTVs on
201505350.1 based on ground-based followup a few months later.

**Comment by timothydmorton** [comment]
Yeah, I was wondering if there were TTVs in 201505350.1--- that would make it look artificially V-shaped.

**Comment by timothydmorton** [comment]
Oh wait-- I just realized I forgot to include the secondary eclipse depth constraints into these calculations.  will update the FPP table when I do.

**Comment by timothydmorton** [comment]
OK, fpp table updated; up to you guys if you want to just have an FPP column in the main table or you want the additional details in a separate FPP table.

**Comment by benmontet** [comment]
I think details in a separate table are appropriate. The final numbers are
important so let's put them in twice, once in the planet properties and
once in the table where they're derived. If the referee doesn't like that
we can change it later.

**Comment by benmontet** [comment]
As someone who thinks about confirmation way more often than I do, what's the current standard FPP you/others use when they declare a planet "confirmed"?

**Comment by dfm** [comment]
So I've _finally_ added a table of physical parameters. It has some formatting issues but will you guys take a look and let me know if you think it needs any big changes?

**Comment by benmontet** [comment]
Okay this looks great to me. Just to verify, which set of stellar params
are you using? I think it's the most conservative ones (case 2 from
yesterday) but want to make sure.

On Thu, Mar 5, 2015 at 3:20 PM, Dan Foreman-Mackey <notifications@github.com

**Comment by dfm** [comment]
That's right. Case 2. I'll make exactly the same table for case 1 and add that as well.

**Comment by benmontet** [comment]
I just got contrast curves from Brendan. I added them to the fpp branch (idl save files, sorry.)

Here's the relevant part of the email below. Will it be possible to update the FPP probabilities for each of these systems? Also, let me know which (if any) would benefit from looking back through archival imaging and seeing if we can rule out non-comoving objevcts.

---

"I finished running the PHARO data for these targets through my pipeline, which registers the images (both saturated and unsaturated dithered frames) and computes contrast curves.  The attached table lists astrometry for stars with bright neighbors.  

PHARO has a weird issue where some (but not all) images have an odd imprint in each quadrant of the detector.  I can even see it in some flats.  Anyway, since it’s not systematic to every image, it’s not removed with bias subtraction or flat fielding in a consistent fashion.  It does affect the contrast curves a little, but it’s mostly cosmetic in nature.  If you do want to make images, I suggest just using the unsaturated frames where the impact is minimal.  I didn’t want to spend the time finding a solution but I can at some point in the future if it’s critical (I’m pretty sure it’s not for these purposes).

The IDL save files have the following arrays:

ANGSCALEARR -> angular scale in arcseconds
ONESIGQLCONPROF -> one sigma contrast curve
FIVE… -> 5-sigma contrast curve
SEVEN… -> I use 7-sigma, but 5-sigma is definitely reasonable (and “believable”)
CORFRACCOVERAGE -> field of view fractional coverage.  Starts at 1.0 and dips to 0 at the
edge of the (square) detector.

So, to make a contrast curve like the one attached, just:

plot, angscalearr, sevensigqlconprof
